script/diff_graphql/run.sh

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env bash
+
+# Diff a rendered page in production when the data source is Content Store vs
+# when the data source is Publishing API's GraphQL endpoint
+
+echo " _____         _           _      _____ _               "
+echo "|     |___ ___| |_ ___ ___| |_   |   __| |_ ___ ___ ___ "
+echo "|   --| . |   |  _| -_|   |  _|  |__   |  _| . |  _| -_|"
+echo "|_____|___|_|_|_| |___|_|_|_|    |_____|_| |___|_| |___|"
+echo "                  _____             _   _____ __        "
+echo "       _ _ ___   |   __|___ ___ ___| |_|     |  |       "
+echo "      | | |_ -|  |  |  |  _| .'| . |   |  |  |  |__     "
+echo "       \_/|___|  |_____|_| |__,|  _|_|_|__  _|_____|    "
+echo "             _                 |_|        |__|          "
+echo "            |_|___                                      "
+echo "            | |   |_ _ _                                "
+echo "            |_|_|_|_|_|_|                     __        "
+echo "     ____  _ ___ ___    ____          _      |  |       "
+echo "    |    \|_|  _|  _|  |    \ ___ ___| |_ _ _|  |       "
+echo "    |  |  | |  _|  _|  |  |  | -_|  _| . | | |__|       "
+echo "    |____/|_|_| |_|    |____/|___|_| |___|_  |__|       "
+echo "                                         |___|          "
+echo "                                                        "
```

Comment:
I had trouble reading this text when I ran the script.  TBH I'm not really sure it adds much value.

Reply:
Yeah, just a bit of fun 😅 happy to remove it!